snappy_pipeline/workflows/somatic_variant_signatures/model.py

```diff
@@ -19,11 +32,31 @@ class SomaticVariantSignatures(SnappyStepModel):
     tools_somatic_variant_annotation: list[str] = []
     """default to those configured for somatic_variant_annotation"""
 
-    filters: list[str] = []
+    has_annotation: bool = False
+    """Needed for building filenames only"""
+
+    filtration_schema: FiltrationSchema = FiltrationSchema.list
+    """Method of variant filtration (if any)"""
+
+    filter_before_annotation: bool = False
+    """Flag if filtration has been done before annotation"""
+
+    filter_sets: Annotated[
+        list[FilterSet], Field(None, deprecated="use `filter_list` instead")
+    ] = []
     """
-    When using variants after the somatic_variant_filtration step,
-    use "no_filter", "dkfz_only", "dkfz_and_ebfilter" or "dkfz_and_ebfilter_and_oxog"
+    DEPRECATED: use `filter_list instead`.
+    Set it to an empty value when using annotated variants without filtration.
```

Comment:
but filter_list is gone?

Reply:
This is similar to the case in the `__init__.py` code: when filtration is done by `sets`, the user can choose her filters at that point. But it is not the case the filtration is done by `list`: the pipeline uses whatever is the filtration output.